bids/layout/tests/test_path_building.py

```diff
@@ -12,9 +12,15 @@ def layout():
 
 def test_bold_construction(layout):
     ents = dict(subject='01', run=1, task='rest', suffix='bold')
-    assert layout.build_path(ents) == "sub-01/func/sub-01_task-rest_run-1_bold.nii.gz"
+    assert layout.build_path(ents, absolute_paths=False) \
+        == "sub-01/func/sub-01_task-rest_run-1_bold.nii.gz", \
+        "Check relative path"
```

Comment:
I think these assert messages will be more obscuring than helpful if the test starts failing. The "absolute_paths=False" should be enough.

I would do:

```Python
relative = Path("sub-01") / "func" / "sub-01_task-rest_run-1_bold.nii.gz"
absolute = Path(layout.root) / relative
assert layout.build_path(ents, absolute_paths=False) == str(relative)
assert layout.build_path(ents, absolute_paths=True) == str(absolute)
# layout fixture created with `absolute_paths=True`, defaulting to absolute
assert layout.build_path(ents) == str(absolute)
```

And then the same after adding the acquisition entity.

This should make at least this test more Windows-friendly. Will require adding `from pathlib import Path`.